service/controller/rolebindingtemplate/resource/rolebinding/create.go

```diff
@@ -49,13 +50,16 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	for _, ns := range template.Status.Namespaces {
 		if !contains(status, ns) {
 			if err = rbac.DeleteRoleBinding(r, ctx, ns, getRoleBindingNameFromTemplate(template)); err != nil {
+				r.logger.Errorf(ctx, err, "Failed to delete roleBinding from namespace %s", ns)
```

Comment:
Same as what Zach said, if you return an error here the rest of the namespaces won't get updated